service/glacier/treehash_test.go

```diff
@@ -61,3 +63,52 @@ func ExampleComputeTreeHash() {
 	// Output:
 	// TreeHash: 154e26c78fd74d0c2c9b3cc4644191619dc4f2cd539ae2a74d5fd07957a3ee6a
 }
+
+func TestComputeHashes(t *testing.T) {
+
+	t.Run("no hash", func(t *testing.T) {
+		var hashes [][]byte
+		tree := glacier.ComputeTreeHash(hashes)
+		if tree != nil {
+			t.Fatalf("expected []byte(nil), got %v", tree)
+		}
+	})
+
+	t.Run("one hash", func(t *testing.T) {
+		hash := sha256.Sum256([]byte("hash"))
+		tree := glacier.ComputeTreeHash([][]byte{hash[:]})
+
+		expected, actual := hex.EncodeToString(hash[:]), hex.EncodeToString(tree)
+		if expected != actual {
+			t.Fatalf("expected %v, got %v", expected, actual)
+		}
+	})
+
+	t.Run("even hashes", func(t *testing.T) {
+		h1 := sha256.Sum256([]byte("h1"))
+		h2 := sha256.Sum256([]byte("h2"))
+		tree := glacier.ComputeTreeHash([][]byte{h1[:], h2[:]})
+		var (
+			expected = "0228c4e26bfc81adb535b2809bdfb1929d9b1cb05c1b2c60a8a4904edbe88ba1"
```

Comment:
just curious, how was this hash produced? in the `one hash` test, `hash` was used as input for both the expected and actual